core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -135,6 +150,18 @@ object Controller {
 
     def optionalProperties = EntitlementProvider.optionalProperties
 
+    private def info(config: WhiskConfig, runtimes: Runtimes) = JsObject(
```

Comment:
Maybe add two entries to the support field, one for git and one for slack. 

Ex: `"support" -> JsObject("slack" -> JsString(slackURL), "git" -> JsString(gitURL))`